core/server/web/api/v2/content/middleware.js

```diff
@@ -1,7 +1,27 @@
-const prettyURLs = require('../../../middleware/pretty-urls'),
+const url = require('url'),
+    debug = require('ghost-ignition').debug('api'),
+    prettyURLs = require('../../../middleware/pretty-urls'),
     cors = require('../../../middleware/api/cors'),
-    urlRedirects = require('../../../middleware/url-redirects'),
-    auth = require('../../../../services/auth');
+    {getBlogRedirectUrl} = require('../../../middleware/url-redirects'),
+    auth = require('../../../../services/auth'),
+    urlService = require('../../../../services/url');
+
+const urlRedirects = (req, res, next) => {
+    const redirectUrl = getBlogRedirectUrl({
```

Comment:
The content API checks for admin redirects, not for blog redirects.

Reply:
Misinterpreted this part `Blog App -> getBlogRedirectUrl and Admin App -> getAdminRedirectUrl.` Though by `Blog App` was meant a `Content API`.  So basically both Admin/Content API's should only use `getAdminRedirectUrl` as implied from removed logic. In this case, the `getBlogRedirectUrl` should stay in `_private` scope, right?

Reply:
👍